### ISSUE TITLE
deb package built on basis of the AppImage tries to overwrite /usr/lib/systemd which is owned by a different package

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/client#12022]

**Comment by DeepDiver1975** [comment]
fixed with https://github.com/owncloud/client/pull/12024

**Comment by saw-jan** [comment]
- no library conflict :heavy_check_mark:  

Tested on Fedora 40 (`ownCloud 5.3.2.15263-rc.5`)

**Comment by saw-jan** [comment]
still error with Fedora versions (with `dbus` installed): It was working :thinking: when I did the above test
```bash
Error: Transaction test error:
  file /usr/lib/systemd/system/dbus.socket from install of owncloud-client-5.3.2.15263_rc.5-1.x86_64 conflicts with file from package dbus-common-1:1.14.10-3.fc40.noarch
```

**Comment by saw-jan** [comment]
Note: no conflict in `ubuntu 22.04` and `ubuntu 20.04`
Works on Debian `11` and `12`

:exclamation: 
The client installation works but when the system is restarted, the system cannot start display-manager and other services due ot clash with system dbus or corrupt dbus files.

**Comment by DeepDiver1975** [comment]
Something went totally wrong - also the deb packages now again contain the systemd files.

**Comment by DeepDiver1975** [comment]
Yeah - I set the tag totally wrong :see_no_evil: - will fix asap ...

**Comment by DeepDiver1975** [comment]
@saw-jan please retest rc.6 - https://github.com/owncloud/client/releases/tag/v5.3.2-rc.6 - THX

**Comment by ishabaral** [comment]
Tested on `Ubutu_24.04`, `Ubuntu_22.04` and `Fedora_41`

- The client installation worked fine.
- Client started and quit properly.
- Restarted the system after installation. No problem found.

**Comment by saw-jan** [comment]
:+1: 
Install report: https://github.com/owncloud/client/issues/12010#issuecomment-2577116226
